idea/idea-frontend-independent/src/org/jetbrains/kotlin/idea/quickfix/AddToStringFix.kt

```diff
@@ -8,28 +8,29 @@ package org.jetbrains.kotlin.idea.quickfix
 import com.intellij.codeInsight.intention.LowPriorityAction
 import com.intellij.openapi.editor.Editor
 import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiFile
 import org.jetbrains.kotlin.idea.KotlinBundle
 import org.jetbrains.kotlin.idea.core.replaced
+import org.jetbrains.kotlin.idea.inspections.KotlinUniversalQuickFix
 import org.jetbrains.kotlin.psi.KtExpression
-import org.jetbrains.kotlin.psi.KtFile
 import org.jetbrains.kotlin.psi.KtPsiFactory
 import org.jetbrains.kotlin.psi.createExpressionByPattern
 import org.jetbrains.kotlin.psi.psiUtil.endOffset
 
-class AddToStringFix(element: KtExpression, private val nullable: Boolean) : KotlinQuickFixAction<KtExpression>(element),
-    LowPriorityAction {
+class AddToStringFix(element: KtExpression, private val useSafeCallOperator: Boolean) :
+    KotlinCrossLanguageQuickFixAction<KtExpression>(element), KotlinUniversalQuickFix, LowPriorityAction {
```

Comment:
What is the purpose of changing the superclasses here? Usually we just change to `KotlinPsiOnlyQuickFixAction`

Reply:
Using `KotlinPsiOnlyQuickFixAction` sounds good. Whatever is used here doesn't seem to matter.